htdocs/public/stripe/ipn.php

```diff
@@ -315,157 +315,178 @@
 	$db->commit();
 } elseif ($event->type == 'payment_intent.succeeded') {		// Called when making payment with PaymentIntent method ($conf->global->STRIPE_USE_NEW_CHECKOUT is on).
 	dol_syslog("object = ".var_export($event->data, true));
-
-	/* TODO LMR We must retreive the invoice and payment amount from the id = ext_payment_id into llx_prelevement_demande
-	include_once DOL_DOCUMENT_ROOT . '/compta/paiement/class/paiement.class.php';
+	include_once DOL_DOCUMENT_ROOT . '/compta/paiement/class/paiement.class.php'; //TOTEST
+	global $stripearrayofkeysbyenv;
+	$errors = 0;
```

Comment:
If it is a counter or code (so not an array), use variable $error   ($errors is used to store an array of error message)